src/components/AllAlerts/index.js

```diff
@@ -2,21 +2,15 @@ import React, { Component } from "react"
 import Row from "react-bootstrap/Row"
 import Col from "react-bootstrap/Col"
 import Container from "react-bootstrap/Container"
-import Accordion from "react-bootstrap/Accordion"
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
-import { faCaretDown } from "@fortawesome/free-solid-svg-icons"
-import classNames from "classnames"
 import PropTypes from "prop-types"
 import styles from "./styles.module.css"
 
 import AlertCard from "../AlertCard/index.js"
 
 import { config } from "@fortawesome/fontawesome-svg-core"
-
+let sev
```

Comment:
I think it's best if this is moved below the `config` statement

Reply:
fixed